INSTALL.md

```diff
@@ -11,16 +11,21 @@ following sections describe them for the supported platforms.
 * GNU Automake
 * GNU Libtool
 * C compiler
-* C Library Development Libraries and Header Files
+
+### C Library Development Libraries and Header Files
```

Comment:
I don't see why you've promoted an item from this list to a section header.

Reply:
The list is confusing so I assumed this was a typo.  I’ll remove it and append “development libraries and header files” to the end of uriparser and gcrypt.